### DISCUSSION TITLE
Does skywalking-eyes support SPDX-short-identifiers?

### DISCUSSION THREAD

**Discussion by tisonkun** [apache/skywalking#7756] [Q&A] [answered]
https://spdx.github.io/spdx-spec/appendix-V-using-SPDX-short-identifiers-in-source-files/

It seems the checker only allows full-text license header.

cc @kezhenxu94 @fgksgf

**Comment by kezhenxu94** [comment, marked as answer]
It's a good feature request. Thanks.

**Reply by tisonkun** [reply]
@kezhenxu94 shall we create an issue on the repository?

**Reply by wu-sheng** [reply]
Already have https://github.com/apache/skywalking/issues/7757

**Reply by tisonkun** [reply]
@wu-sheng interesting, the link information is hard to find as it is not in the timeline but in the corner.

![image](https://user-images.githubusercontent.com/18818196/134263682-53d954db-e793-41c8-86b2-a61fa20bdcd5.png)

**Reply by wu-sheng** [reply]
Yes, it is what GitHub discussion looks like today.
